src/main/java/storagesystem/viewcontroller/inventory/DetailedItemViewController.java

```diff
@@ -0,0 +1,359 @@
+package storagesystem.viewcontroller.inventory;
+
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.Event;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.*;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Pane;
+import javafx.stage.FileChooser;
+import storagesystem.model.*;
+import storagesystem.services.PictureHandler;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Controls a detailed view of an item. Can be used to book an item.
+ *
+ * @author Jonathan Eksberg, Carl Lindh
+ */
+
+//todo refactor and make code good looking.
+public class DetailedItemViewController extends AnchorPane {
+    private final IReservable reservableItem;
+    private final Team itemOwner;
+    private List<Location> locationList;
+    ObservableList<String> locationNames;
+
+    @FXML
+    private AnchorPane contentPane;
+    @FXML
+    private ImageView itemPageImageView;
+    @FXML
+    private ImageView closeButtonImageView;
+    @FXML
+    private TextArea itemPageNameTA;
+    @FXML
+    private Label itemPageIDLabel;
+    @FXML
+    private Label itemPageTeamOwnerLabel;
+    @FXML
+    private Slider itemPageConditionSlider;
+    @FXML
+    private TextArea itemPageDescriptionTA;
+    @FXML
+    private TextArea itemPageUserRequirementsTA;
+    @FXML
+    private ChoiceBox isReservableChoiceBox;
+    @FXML
+    private TextArea itemPageAmountTA;
+    @FXML
+    private ChoiceBox itemPageLocationChoicebox;
+
+
+    @FXML
+    Button itemPageReserveBtn;
+    @FXML
+    Button itemPageSaveButton;
+    @FXML
+    private Pane editPane;
+
+    private PictureHandler pictureHandler = new PictureHandler();
+
+    DetailedItemViewController(IReservable reservableItem, Team itemOwner) {
+        this.reservableItem = reservableItem;
+        this.itemOwner = itemOwner;
```

Comment:
Should it not instead take only item as param, and ask the modell for the owner?